### PR TITLE
Bugfix: fix some yurtctl errors

### PR DESCRIPTION
**Pull request by wenjun93**

### CODE REVIEW

File: pkg/yurtctl/util/kubernetes/util.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ var (
 	// PropagationPolicy defines the propagation policy used when deleting a resource
 	PropagationPolicy = metav1.DeletePropagationForeground
 	// WaitServantJobTimeout specifies the timeout value of waiting for the ServantJob to be succeeded
-	WaitServantJobTimeout = time.Minute * 2
+	WaitServantJobTimeout = time.Minute * 5
```
Comment (Fei-Guo) [inline pkg/yurtctl/util/kubernetes/util.go:59]: Why do you increase the timeout?

Comment (wenjun93) [reply pkg/yurtctl/util/kubernetes/util.go:59]: In my test env, 2min is too short for the job finish , however, after timeout, the convert job succeeded

Comment (wenjun93) [reply pkg/yurtctl/util/kubernetes/util.go:59]: sure, thanks

File: pkg/yurtctl/cmd/convert/convert.go [hunk 2]
```diff
@@ -198,12 +201,20 @@ func (co *ConvertOptions) RunConvert() (err error) {
 	klog.V(4).Info("the server version is valid")
 
 	// 2. label nodes as cloud node or edge node
-	nodeLst, err := co.clientSet.CoreV1().Nodes().List(metav1.ListOptions{})
+	var nodeList *v1.NodeList
+	var requirement *labels.Requirement
+	// skip master node
```
[the remark this answers is not in the document]
Comment (wenjun93) [reply pkg/yurtctl/cmd/convert/convert.go:206]: seems the nodelist is not used for tunnel deploy

[the remark this answers is not in the document]
Comment (charleszheng44) [reply pkg/yurtctl/cmd/convert/convert.go:206]: The nodelist here, will be used to add the cloud/edge node label to each node. If we skip the master node, then the master node will not be marked as the cloud node while the `yurt-tunnel-server` can only be deployed on the cloud node.

**Comment by charleszheng44** [comment]
LGTM

**Pull request merged**